### ISSUE TITLE
No preview available after container restart

### ISSUE THREAD

**Issue by vanlueckn** [pixelfed/pixelfed#3885]
Hi,

i am using pixelfed in a dockerized environment. 
I am using a s3 storage for my images.

The images get uploaded to s3.

When i restart the container instead of the images only "No preview available" is shown in pixelfed.

Horizon is working.

**Comment by syrabo** [comment]
I save the pictures local; not container; installation at plesk with Ubuntu 2022.04 with manuals: https://lab.uberspace.de/guide_pixelfed/ and  https://docs.pixelfed.org/running-pixelfed/installation/

I also had this problem. 

After researching setting the permissions in the storage folder, I found that the folders with the images were set to not readable for the group.

The following solution helped me: 

https://laracasts.com/discuss/channels/laravel/storage-folder-is-not-showing-in-the-public-folder-after-linking-the-path

I set the folder: FILESYSTEM_DRIVER=public

this was set to local.

Maybe this helps here too; I think it should be documented because local is set like this in the .env template.
